jvb/src/main/kotlin/org/jitsi/videobridge/cc/av1/Av1DDAdaptiveSourceProjectionContext.kt

```diff
@@ -228,6 +228,9 @@ class Av1DDAdaptiveSourceProjectionContext(
 
     private fun frameIsNewSsrc(frame: Av1DDFrame): Boolean = lastAv1FrameProjection.av1Frame?.matchesSSRC(frame) != true
 
+    private fun frameIsProjectable(frame: Av1DDFrame): Boolean =
+        frameIsProjectable(frame) || frame.index >= lastFrameNumberIndexResumption
```

Comment:
This doesn't seem right. Does it need to call `frameIsNewSsrc` like vp9?

Reply:
Damian also caught this...autocomplete error.